src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -60,23 +61,52 @@ class ProfileWrapper extends Component {
     </div>
   );
 
+  createBreadCrumbAttributes = () => {
+    const { location, route } = this.props;
+    const activeLocation = location?.pathname.replace('/', '');
+    const childRoutes = route?.childRoutes;
+    const activeRoute = childRoutes.find(
+      childRoute => childRoute.path === activeLocation,
+    );
+
+    const activeRouteName = activeRoute?.name;
+
+    return { activeLocation, activeRouteName };
+  };
+
   // content to show after data has loaded
   // note that `children` will be passed in via React Router.
-  mainContent = () => (
-    <>
-      <MobileMenuTrigger />
-      <div className="mobile-fixed-spacer" />
-      <ProfileHeader />
-      <div className="usa-grid usa-grid-full">
-        <div className="usa-width-one-fourth">
-          <ProfileSideNav />
-        </div>
-        <div className="usa-width-two-thirds vads-u-padding-bottom--4 vads-u-padding-x--1 medium-screen:vads-u-padding--0 medium-screen:vads-u-padding-bottom--6">
-          {this.props.children}
+  mainContent = () => {
+    const {
+      activeLocation,
+      activeRouteName,
+    } = this.createBreadCrumbAttributes();
+
+    return (
+      <>
+        {/* Breadcrumbs */}
+        <Breadcrumbs className="vads-u-padding-x--1 vads-u-padding-y--1p5 medium-screen:vads-u-padding-y--0">
+          <a href="/">Home</a>
+          <a href="/profile-2/">Profile</a>
+          <a href={activeLocation}>{activeRouteName}</a>
```

Comment:
I _think_ these could both be `Link`s to enable the client-side routing like https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/components/Breadcrumbs.jsx#L20 - although I'm not sure if you already receive that through a regular `<a>`